packages/pangea-node-sdk/src/services/vault.ts

```diff
@@ -31,32 +31,25 @@ class VaultService extends BaseService {
    * @summary State change
    * @description Change the state of a specific version of a secret or key.
    * @operationId vault_post_v1_state_change
```

Comment:
```suggestion
   * @operationId vault_post_v2_state_change
```
(and the rest below)

Reply:
Fixed